lib/model/content.dart

```diff
@@ -855,7 +855,9 @@ class _ZulipContentParser {
 
     if (localName == 'a'
         && (className.isEmpty
-            || (className == 'stream-topic' || className == 'stream'))) {
+            || (className == 'stream-topic'
+                || className == 'stream'
+                || className == 'message-link'))) {
```

Comment:
nit:
```suggestion
            || className == 'stream-topic'
            || className == 'stream'
            || className == 'message-link')) {
```

(I think the extra parens are a legacy of how the logic used to be more complicated with `classes`; introduced in 3cd84bb97a3661ecc464b6227040aa0e9a10e0e0, simplified in 4c945477badbbad050eacea0322da2c326a2a92d but could have been simplified a bit further.)